repository_service_tuf/cli/admin/import_targets.py

```diff
@@ -30,23 +31,33 @@ def _check_csv_files(csv_files: List[str]):
 
 
 def _parse_csv_data(
-    csv_file: str, succinct_roles: SuccinctRoles
+    db: Any,
```

Comment:
Can you improve the annotation to be more precise?
From what I see `db` is a `sqlaclehmy` object called `Connection`.

Reply:
See[ my comment above](https://github.com/repository-service-tuf/repository-service-tuf-cli/pull/304#discussion_r1246452172).